servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/EraseRepository.java

```diff
@@ -48,6 +48,11 @@ public class EraseRepository extends BaseCommand {
   protected Integer callWithPersist() {
     warnOnInMemory();
 
+    if (!repositoryLogic(persist).repositoryExists()) {
+      spec.commandLine().getErr().println("Nessie repository does not exist");
+      return 1;
```

Comment:
Maybe use a different exit code in case shell tools need to distinguish this failure mode from error that happen during the actual repo deletion process?